static/js/composebox_typeahead.js

```diff
@@ -792,7 +794,15 @@ const show_flatpickr = (element, callback, default_timestamp) => {
     });
     instance.open();
     container.find(".flatpickr-monthDropdown-months").trigger("focus");
-};
+
+    $(".flatpickr-calendar").on("keydown", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        // TODO: This event (e) should be forwarded to the flatpickr instance
+        //       and act correctly
+    });
+    return instance;
```

Comment:
I dropped this hunk, since I suspect it doesn't do what we want; I've opened https://github.com/zulip/zulip/issues/20130 for the keyboard UI problem.  It'd be a good thing for you to work on next, but if you're busy let me know and I can try to find a different volunteer :).